devel/push

```diff
@@ -5,14 +5,25 @@
 # By default this publishes the "latest" tag, but you can provide other tags in
 # addition to or instead of "latest".
 #
+# Errors if a tagged image has already been pushed.
+#
 set -euo pipefail
 
 # If no tags are given, default to "latest".
 if [[ $# -eq 0 ]]; then
     set -- latest
 fi
 
+BASE_IMAGE="nextstrain/base"
+BASE_BUILDER_IMAGE="nextstrain/base-builder"
+
 for tag in "$@"; do
-    docker push nextstrain/base-builder:$tag
-    docker push nextstrain/base:$tag
+
+    if [[ $(docker image inspect --format "{{.RepoDigests}}" $BASE_IMAGE:$tag) != '[]' || $(docker image inspect --format "{{.RepoDigests}}" $BASE_BUILDER_IMAGE:$tag) != '[]' ]]; then
+        echo "At least one of the tagged images has already been pushed. This can happen if the newly built image is not available in the local registry." >&2
+        exit 1
+    fi
```

Comment:
[CI failed post-merge on this error](https://github.com/nextstrain/docker-base/runs/7984133243?check_suite_focus=true). Will investigate.

Reply:
Fixing in #79.